spec/services/message_employee_matcher_spec.rb

```diff
@@ -2,10 +2,18 @@
 
 describe MessageEmployeeMatcher do
   describe "#run" do
+    before do
```

Comment:
I prefer inlining `before` and `after`  actions to avoid the mystery guest problem - it's more duplication, but gives more transparency about the behavior of each spec. What do you think about doing that in this file?